### DISCUSSION TITLE
Praise

### DISCUSSION THREAD

**Discussion by Excalibur025** [akinomyoga/ble.sh#151] [General]
After using the zsh autocomplete I looked everywhere for a suitable feature that worked in bash and found your code. I love it, it's amazing and drastically helps my productivity in the CLI. Thank you for creating this!

**Comment by akinomyoga** [comment]
Thank you!

**Reply by Knusper** [reply]
I also chime in.  This is really a piece of art!   I start enjoy using bash again...

**Reply by akinomyoga** [reply]
Thank you! These words encourage me. If you have any suggestions that may improve `ble.sh`, please feel free to open issues!

**Comment by akinomyoga** [comment]
I remembered that I have actually received a similar issue #27 in the past. At that time, there was no "Discussion" in the GitHub free plan, so the issue was finally closed a few weeks later, but I think it's worth mentioning it here. Anyway, thank you for your encouraging words!
